go.mod

```diff
@@ -11,7 +11,7 @@ require (
 	github.com/celestiaorg/go-fraud v0.2.1
 	github.com/celestiaorg/go-header v0.6.2
 	github.com/celestiaorg/go-libp2p-messenger v0.2.0
-	github.com/celestiaorg/go-square v1.1.0
+	github.com/celestiaorg/go-square v1.1.1-0.20241007084659-3f4910615eeb
```

Comment:
will be changed once we merge https://github.com/celestiaorg/go-square/pull/108 and make a new release

Reply:
e9f6cf7